### DISCUSSION TITLE
Can't Log Into Vercel on Chromebook

### DISCUSSION THREAD

**Discussion by osseonews** [vercel/vercel#6818] [Help]
It's impossible to log into Vercel on Chromebook. We have tried everything. Cleared cookies, restart machine etc. Chromebook is fully updated to latest version. Console shows "uncaught syntax error: unexpected token" in one of the static JS files. Have anyone else had this issue? Actually, none of the menu, including the Login link even shows on the website in chromebook, we have to go the /login page directly. But then none of the logins presented are actually clickable. The buttons simply don't do anything when you click on them. Vercel is the only website we experience a problem with.

**Comment by fu-sen** [comment]
I'm using CloudReady and [Chromium OS for Raspberry Pi](https://github.com/FydeOS/chromium_os-raspberry_pi), I can browse the Vercel Dashboard successfully.
So this is a problem specific to your environment.
For example, does your Chrome use AD block? It may behave unintentionally.
